ToolBox/QuantConnect.ToolBox.csproj

```diff
@@ -368,6 +369,9 @@
     <None Include="CoarseUniverseGenerator\config.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Include="config.json">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
Generally you want this set to `PreserveNewest` so if you have a `config.json` in your `bin/Debug` directory that you've been editing, a new build won't overwrite it w/out warning

Reply:
Thank you, I am changing this with separate commit - as well as changing the name of toolbox config file to 'toolboxoptions.json' (isn't it very complicated name?) As some downloaders may have used Lean settings config.json file and keep it in bin/Debug:
https://github.com/QuantConnect/Lean/blob/master/ToolBox/YahooDownloader/YahooDownloaderProgram.cs#L44
to avoid the conflict in namings.